### PR TITLE
docs: add Chinese doc for discovery DNS method

### PR DESCRIPTION
**Pull request by Singularity0909**
### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/zh/latest/discovery/dns.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+* [通过 DNS 服务发现](#通过 DNS 服务发现)
```
Comment (tokers) [inline docs/zh/latest/discovery/dns.md:24]:
```suggestion
* [基于 DNS 的服务发现](#基于 DNS 的服务发现)
```

File: docs/zh/latest/discovery/dns.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    * [SRV 记录](#SRV 记录)
+
+## 通过 DNS 服务发现
```
Comment (tokers) [inline docs/zh/latest/discovery/dns.md:27]: Ditto.

File: docs/zh/latest/discovery/dns.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+```yaml
+# 添加到 config.yaml
+discovery:
+   dns:
+     servers:
+       - "127.0.0.1:8600"          # 使用 DNS 服务器的真实地址
+```
+
+与配置 Upstream `nodes` 项不同的是，DNS 服务发现将返回所有的记录。例如按照以下的 upstream 配置：
```
Comment (tokers) [inline docs/zh/latest/discovery/dns.md:41]:
```suggestion
与在 Upstream的 `nodes` 对象中配置域名不同的是，DNS 服务发现将返回所有的记录。例如按照以下的 upstream 配置：
```

File: docs/zh/latest/discovery/dns.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+    "nodes": [
+        {"host": "1.1.1.1", "weight": 1},
+        {"host": "1.1.1.2", "weight": 1}
+    ]
+}
+```
+
+注意所有来自 `test.consul.service` 的 IP 都有相同的权重。
+
+如果一个服务同时有 A 和 AAAA 记录，那么 A 记录的优先级更高。
```
Comment (tokers) [inline docs/zh/latest/discovery/dns.md:67]: It seems we should also specify the precedence between `A` and `SRV`.

Comment (Singularity0909) [reply docs/zh/latest/discovery/dns.md:67]: I would appreciate it if you make supplement on the document in English before I update it?

https://github.com/apache/apisix/blame/master/docs/en/latest/discovery/dns.md#L69

**Review by tokers — APPROVED** [review]

File: docs/zh/latest/discovery/dns.md (hunk 4 above)
Comment (spacewander) [reply docs/zh/latest/discovery/dns.md:67]: I just make a PR for you:
https://github.com/apache/apisix/pull/5033/files

**Review by tzssangglass — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**